prime-dotnet-webapi/Models/Certifications/UnlistedCertification.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace Prime.Models
+{
+    [Table("UnlistedCertification")]
+    public class UnlistedCertification : BaseAuditable, IEnrolleeNavigationProperty
+    {
+        [Key]
+        public int Id { get; set; }
+        [JsonIgnore]
+        public int EnrolleeId { get; set; }
+        public string UnlistedCollegeName { get; set; }
+        public string UnlistedCollegeCode { get; set; }
+        public DateTimeOffset? UnlistedRenewalDate { get; set; }
+        public Enrollee Enrollee { get; set; }
```

Comment:
Do you need "Unlisted" as part of the field names, considering the class is named `UnlistedCertification`?